### PR TITLE
MAINT: centralize, upgrade es_version and gradle version

### PR DESCRIPTION
**Pull request by chenqi0805**
*Issue #, if available:*

*Description of changes:*
This PR adds es_version into versionMap and upgrade it to 7.10.2. Correspondingly, gradle version has to be upgraded to 6.6.1.

Note: tested with the sample app in examples/dev

-----

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: build-resources.gradle [hunk 1]
```diff
@@ -1,6 +1,7 @@
 //preferably try to main the alphabetical order
 ext.versionMap = [
-        opentelemetry_proto    : '0.10.0'
+        opentelemetry_proto    : '0.10.0',
+        es_version: '7.10.2'
```
Comment (wrijeff) [inline build-resources.gradle:4]: Can this be upgraded to 7.10.3?

**Review by wrijeff — APPROVED** [review]
👍 

~~Just as a sanity check, could you try the /dev trace sample app setup with these changes?~~

File: build-resources.gradle (hunk 1 above)
Comment (chenqi0805) [reply build-resources.gradle:4]: Confirmed that 7.10.3 should be 7.10 in ES branches: https://github.com/elastic/elasticsearch/blob/e65acafcd6347ae5cdd8df1d344769bd237ff88c/server/src/main/java/org/elasticsearch/Version.java#L155-L156

Will replace and check again.

Comment (chenqi0805) [reply build-resources.gradle:4]: Well...actually 7.10 is not a release branch so the closest we have is still 7.10.2.

**Review by kowshikn — APPROVED** [review]

**Pull request merged**